docs/src/distributions/normal.md

```diff
@@ -30,7 +30,6 @@ Normal{T}
 
 ```@docs
 Normal(μ::AbstractVector, Σ::CovarianceParameter)
-Normal(μ::AbstractVector, Σ::AbstractMatrix)
```

Comment:
Is there no AbstractMatrix constructor anymore then?

Reply:
Nope.